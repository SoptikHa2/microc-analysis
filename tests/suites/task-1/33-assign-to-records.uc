 main() {
   var r,s;
   r = { a: null, b: null };
   s = { c: 1 };
   r.a = &s;
   r.b = &s;
   s.c = 2;
   return (*(r.a)).c + (*(r.b)).c;
 }
