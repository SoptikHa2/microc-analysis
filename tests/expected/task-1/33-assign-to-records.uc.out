---
test: run suites/task-1/33-assign-to-records.uc
---

exit: 4
